osgar/explore.py

```diff
@@ -85,6 +87,8 @@ def follow_wall_angle(laser_data, radius, right_wall=False):
         found_countinuation = False
         for i in range(last_wall_idx + 1, size):
             dist = distances[i]
+            if dist > max_wall_distance or distance == 0:
```

Comment:
This line should contain `dist` instead of `distance`.

Reply:
thanks for hint! could this be related to R2 behavior on start area too?

Reply:
The good news is that is should have influence, i.e. original replay asserts (after the change)
```
AssertionError: ([300, 7020], [300, -14040], datetime.timedelta(0, 41, 517384))
```
... I am also wondering if to unify it with previous cycle, which uses enumerate and distance ... but step by step.
thanks

Reply:
`ver103fall4` (`4145b82e-23c0-45eb-a2e1-a2187faab40d`) - locally R2 entered the tunnel ... so we will see this time